packages-exp/app-exp/src/public-types.ts

```diff
@@ -70,16 +70,48 @@ export interface FirebaseApp {
 /**
  * @public
  *
- * Firebase configuration object
+ * Firebase configuration object. Contains a set of parameters required by
+ * services in order to successfully communicate with Firebase server APIs
+ * and to associate client data with your Firebase project and
+ * Firebase application.
```

Comment:
Should we say somewhere that they can get this entire config from the console? Or something so nobody makes the mistake of thinking they have to think about and populate each field individually.

Reply:
Good point, done.